MAINTAINERS_GUIDE.md

```diff
@@ -0,0 +1,8 @@
+# `c8` Maintainers Guide
+
+- Label the issues appropriately, see the list of labels and their description [here](https://github.com/bcoe/c8/labels)
+- Be welcoming to first-time contributors, identified by the GitHub `first-time contributor` badge.
+- At least two Collaborators must approve a pull request before the pull request lands.
```

Comment:
Just curious if @bcoe thinks this project is at that level of maturity already... I wouldn't want to block a contribution waiting for that 2nd review.

Maybe this could be relaxed to `should be approved by 1-2 collaborators before merging`? 🤷‍♂️